### PR TITLE
Fix Tsitu Auto-Loader not auto-loading

### PR DESCRIPTION
**Pull request by hymccord**
- Change devtool for dev since eval() is CSP blocked
- Fix for always showing Tsitu Auto-Loader

### CODE REVIEW

**Comment by hymccord** [comment]
As a side-effect (a good one) of using the same function for showing the loader, it will now use the offset in the setting for showing when clicked from the popup menu.

**Comment by hymccord** [comment]
I'm debugging some funniness with the loader_offset having problems if it's a number. It wants it to be a string (bm-menu uses `.concat` on it) but everywhere we deal with it on our side it's a number.

**Comment by DevJackSmith** [comment]
Interesting find. I'm not sure if it mattered because we had to save settings in order to enable tsitus autoloader, or we just didn't notice first time and adjusted right away. Either way that's a good find.

File: src/scripts/main.js [hunk 1]
```diff
@@ -2819,6 +2819,11 @@
             tempversion = "TEST version";
         }
         window.console.log("MHCT: " + tempversion + " loaded! Good luck!");
+
+        // Tell content script we are done loading
+        window.postMessage({
+            mhct_finish_load: 1,
+        });
```
[the remark this answers is not in the document]
Comment (DevJackSmith) [reply src/scripts/main.js]: done

**Comment by DevJackSmith** [comment]
looks good, tested, it works for both the autoloader and the maps (at least locally).

**Review by DevJackSmith — APPROVED** [review]

**Pull request merged**